tck/pom.xml

```diff
@@ -440,6 +441,7 @@
                             <goal>single</goal>
                         </goals>
                         <configuration>
+                        	<skip>${skipAssembly}</skip>

```

Comment:
Does it work? It seems the parameter is [skipAssembly](https://maven.apache.org/plugins/maven-assembly-plugin/single-mojo.html#skipAssembly).